frontend/src/layout/navigation-3000/sidepanel/sidePanelLogic.tsx

```diff
@@ -70,10 +72,6 @@ export const sidePanelLogic = kea<sidePanelLogicType>([
                         return false
                     }
 
-                    if (tab === SidePanelTab.Activation && (!isReady || hasCompletedAllTasks)) {
-                        return false
-                    }
-
```

Comment:
I'm a bit unsure about this change. Essentially I'm moving removing the activation tab from `enabledTabs`, instead of `visibleTabs`. Maybe @benjackwhite can say if we'd still need that.